modules/form-builder/main/index.coffee

```diff
@@ -21,6 +20,10 @@ class Form extends hx.EventEmitter
     selection = entry.append(nodeType).attr("id",id)
     extras = f.call(selection) or {}
 
+    # Define the default function for enabling/disabling a form property
+    extras.disable ?= (sel, disable) ->
+      sel.attr('disabled', if disable then true else undefined)
```

Comment:
Not sure what the quotes add apart from two extra characters? It still resolves to `disabled="true"` on the element.
 
Also, I'd normally put `if disable then '' else undefined` because the disabled attribute works on existence not value (e.g. if it has the disabled attribute, it's disabled regardless of the value added) but I thought having `true` was more explicit as to what this is doing.